services/apps/test.properties

```diff
@@ -1,44 +1,33 @@
-# Connection details.
```

Comment:
The test properties file was outdated. This change just brings it up-to-date with the current configuration.